setup_scripts/install_dependencies.sh

```diff
@@ -70,6 +70,12 @@ sudo apt-get install -y\
     clang-format\
     python-rosinstall
 
+echo "================================================================"
+echo "Installing Robotic Arm Dependencies
+echo "================================================================"
+
+sudo ./snowbots_arm_deps/install-snowbots-arm-dep.sh
```

Comment:
```suggestion
./snowbots_arm_deps/install-snowbots-arm-dep.sh
```
All the commands in this `install-snowbots-arm-dep.sh` script that require elevated privileges are already prefixed with `sudo`.